packages/frontend/src/components/common/TransferItemsModal/TransferItemsModal.tsx

```diff
@@ -22,77 +27,165 @@ const TransferItemsModal = <
+                            </Button>
+                        ) : (
+                            <Button
+                                disabled={!spaceUuid}
+                                onClick={() => {
+                                    if (spaceUuid) {
+                                        onConfirm(spaceUuid);
+                                    }
+                                }}
+                            >
+                                Confirm
+                            </Button>
+                        )}
                     </Group>
-                </Group>
+                </>
             }
         >
-            <Text fz="sm" fw={500}>
-                Select a space to transfer {items.length > 1 ? 'items' : 'item'}{' '}
-                to:
-            </Text>
-
-            <Paper
-                component={ScrollArea}
-                w="100%"
-                h="320px"
-                withBorder
-                px="sm"
-                py="xs"
-            >
-                <Tree
-                    data={spaces}
-                    value={spaceUuid}
-                    onChange={setSpaceUuid}
-                    topLevelLabel="Spaces"
-                />
-            </Paper>
-
-            {selectedSpaceLabel ? (
-                <Alert color="gray">
-                    <Text fw={500}>
-                        Transfer {items.length}{' '}
-                        {items.length > 1 ? 'items' : 'item'} to{' '}
-                        {selectedSpaceLabel}.
+            {isCreateNewSpace ? (
+                <>
+                    <Box>
+                        <Button
+                            variant="subtle"
+                            leftIcon={<IconArrowLeft size={14} />}
+                            onClick={createNewSpaceHandlers.close}
+                        >
+                            Back
+                        </Button>
+                    </Box>
+
+                    <Text fz="sm" fw={500}>
+                        You are creating a new space in{' '}
+                        <Text span fw={600}>
+                            '{selectedSpaceLabel}'
+                        </Text>
+                    </Text>
+
+                    <TextInput
+                        label="Name"
+                        placeholder="Space name"
+                        required
+                        disabled={createSpaceMutation.isLoading}
+                        value={newSpaceName}
+                        onChange={(e) => setNewSpaceName(e.target.value)}
+                    />
+
+                    <Alert color="blue" icon={<IconInfoCircle size={16} />}>
+                        <Text fw={500} color="blue">
+                            Permissions will be inherited from{' '}
+                            <Text span fw={600}>
+                                '{selectedSpaceLabel}'
+                            </Text>
+                        </Text>
+                    </Alert>
```

Comment:
would be useful to include this alert for both scenarios
When moving to a nested space, you can display this IMO

Reply:
good point! I'll make the change now